modules/core/src/lib/deck.js

```diff
@@ -474,9 +432,11 @@ export default class Deck {
       }
     });
 
-    if (this.controller) {
-      this.controller.setProps({eventManager: this.eventManager});
-    }
+    this.viewManager = new ViewManager({
```

Comment:
Nit: Can it be created earlier and the event manager added here? Assuming that gets too fiddly...

Reply:
`this.viewManager.setView` gets called before this. So either in Deck or in ViewManager we need to do the empty check.